src/helpers/execHelper.ts

```diff
@@ -16,7 +16,7 @@ export const exec = (argv: string[]): Promise<void> => {
     }
 
     const p = proc.spawn(command, argv.slice(1), {
-      shell: true,
+      shell: false,
```

Comment:
It looks like this may have some negative effect on the [Acceptance Tests](https://github.com/saml-to/cli/actions/runs/6011785292/job/16329908711).

Is flipping this necessary to make Powershell work? If so, we may have to add an OS-level conditional here then.

FYI, The acceptance tests it uses a [special combination of `--headless` and `--withToken {GITHUB_TOKEN}`](https://github.com/saml-to/cli/blob/main/.github/workflows/acceptance-tests.yml#L105), would you mind:
  - using your token in `$HOME/.saml-to/github-token.json` (or any PAT) and using the `--withToken` flag in your verifications on your system and report back?

Reply:
I tired everything else I knew and this it.  Upon googling every bit of information I can find says overall, shell:True is bad practice.

We can probaby avoid this siwht some OS specific code (check the platform, change the shell value) I don't think to explain why that is less than ideal to you but it's an option.

I will do that tomorrow you want that on 2.1.1?

Reply:
By the way what shell True does on windows is append the executing binary to the command, that is why you see C:\Progra m.  Another thought it was figure out how to pop off those entries till we hit what want.